sdcm/utils/common.py

```diff
@@ -58,11 +58,9 @@
 from mypy_boto3_ec2 import EC2Client, EC2ServiceResource
 from mypy_boto3_ec2.service_resource import Image as EC2Image
 import docker  # pylint: disable=wrong-import-order; false warning because of docker import (local file vs. package)
-import libcloud.storage.providers
-import libcloud.storage.types
-from libcloud.compute.base import Node as GCENode, NodeImage as GCEImage
-from libcloud.compute.providers import get_driver
-from libcloud.compute.types import Provider
+from google.cloud.storage import Blob
```

Comment:
Since it is located in the `common` module I think it is better to rename `Blob` to something more specific.
For example:
```from google.cloud.storage import Blob as GceStorageBlob```

Reply:
changed it (even this is was used only to get tab compilation on the blob)